src/packages/@ncigdc/modern_components/ConsequencesTable/ConsequencesTable.js

```diff
@@ -130,9 +130,11 @@ export default compose(
       canonicalTranscriptId,
       functionalImpactTranscript,
       theme,
+      context,
     }: TProps = {},
   ) =>
     <LocalPaginationTable
+      entityName={context}
```

Comment:
Instead of passing this down, the default of `projects` should probably be removed from `LocalPaginationTable`. That was my bad, doesn't really make sense to have a default there :)